scripts/sysman/engine.yml

```diff
@@ -139,6 +160,29 @@ params:
       name: pStats
       desc: "[in,out] Will contain a snapshot of the engine group activity counters."
 --- #--------------------------------------------------------------------------
+type: function
```

Comment:
@sanilkumar0 we also have this as part of PR #134 right?

Reply:
so why do we need to add this in this PR? it would cause merge conflicts right?

Reply:
shoot.. I will revert it.. not sure how this change came into PR#74